irene-perez/challenge/todolist-redux/todo-list/src/components/TaskList.jsx

```diff
@@ -0,0 +1,31 @@
+import React from 'react';
+import { PropTypes } from 'prop-types';
+import { connect } from 'react-redux';
+import TASKS from '../constant/task-mock';
+
+function TaskList() {
+  return (
+    <ul>
+      {TASKS.map((task) => (
+        <li>
```

Comment:
When you do a loop to render components you need to provide a key. In this case you should assign a key for evert <li>, for example: <li. className = {task.id} >

Reply:
this should be <li. **key** = {task.id} >

Reply:
okey :)